gensim/models/doc2vec.py

```diff
@@ -142,16 +173,33 @@ def train_document_dm(model, doc_words, doctag_indexes, alpha, work=None, neu1=N
                        model.vocab[w].sample_int > model.random.rand() * 2**32]
 
         for pos, word in enumerate(word_vocabs):
-            reduced_window = model.random.randint(model.window)  # `b` in the original doc2vec code
+            reduced_window = model.random.randint(
+                model.window)  # `b` in the original doc2vec code
```

Comment:
This seems a good example of where strict 79-char line-lengths impair readability. (I'd almost rather lose the comment than move a tiny atomic parameter to a new line!)